src/identity-utils.ts

```diff
@@ -17,74 +30,101 @@ export type IParseCachedIdentityResponse = (
     identityMethod: string,
     knownIdentities: IKnownIdentities,
     fromCachedIdentity: boolean
-) => void
+) => void;
 
 export interface IKnownIdentities extends UserIdentities {
     device_application_stamp?: string;
 }
 
 export interface ICachedIdentityCall {
+    // https://go.mparticle.com/work/SQDSDKS-6672
     responseText: string;
     status: number;
     expireTimestamp: number;
 }
 
+// https://go.mparticle.com/work/SQDSDKS-6568
```

Comment:
I'm not convinced this ticket clearly explains what should happen to `xhrIdentityResponseAdapter` during the clean up.  Consider adding to the description so that it is clear.